alpha/nodejs-chart/README.md

```diff
@@ -0,0 +1,54 @@
+# Node.js Helm Chart
+A Helm chart for building and deploying a [Node.js](https://nodejs.org/) application on OpenShift.
+
+## Prerequisites
+You will need to create a pull secret if you pull an S2I builder or Docker base image from an external registry. Use the following command as a reference to create your pull secret:
+```bash
+oc create secret docker-registry my-pull-secret --docker-server=$SERVER_URL --docker-username=$USERNAME --docker-password=$PASSWORD --docker-email=$EMAIL
+```
+
+You can use this secret by passing `--set build.pullSecret=my-pull-secret` to `helm install`, or you can configure this in a values file:
+```yaml
+build:
+  pullSecret: my-pull-secret
+```
+and apply by passing `--values $VALUES_FILE`.
+
+## Values
+This section describes the Values used to configure this chart.
+
+Below is a table of each value used to configure this chart.
+
+| Value | Description | Default | Additional Information |
+| ----- | ----------- | ------- | ---------------------- |
+| `image.name` | Name of the image you want to build/deploy | Defaults to the Helm release name. | The chart will create/reference an [ImageStream](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html) based on this value. |
+| `image.tag` | Tag that you want to build/deploy | `latest` | The chart will create/reference an [ImageStreamTag](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html#images-using-imagestream-tags_image-streams-managing) based on the name provided |
+| `build.enabled` | Determines if build-related resources should be created. | `true` | Set this to `false` if you want to deploy a previously built image. Leave this set to `true` if you want to build and deploy a new image. |
+| `build.uri` | Git URI that references your git repo | https://github.com/nodeshift-starters/nodejs-rest-http | This value defaults to a sample application. Be sure to override this if you want to build and deploy your own application. |
```

Comment:
@sbose78 Do we want to move this quickstart under https://github.com/redhat-developer-helm-quickstarts to be consistent with Quarkus?

Reply:
Given that https://github.com/nodeshift-starters is a popular location where repos are maintained by Red Hat, we may leave it there for now.

Reply:
+1 on pointing to the example where it is. If `redhat-developer-helm-quickstarts` is going to hold a number of examples at some point we can considered putting a quick start specifically for helm using Node.js later on.